content/plugin/log/filtering.mdx

```diff
@@ -0,0 +1,92 @@
+| `TF_LOG_CORE` | Terraform binary | Does not include providers. |
+| `TF_LOG_PROVIDER` | All providers used during the run | Overrides logging environment variables for specific providers (e.g., `TF_LOG_PROVIDER_AWS`) and provider SDKs (e.g., `TF_LOG_PROTO`).|
+| `TF_LOG_PROVIDER_{PROVIDER_NAME}` (e.g., `TF_LOG_PROVIDER_AWS`)| Specific provider  | Only applies to providers that write logs using the `tflog` package. Does not affect logs written by legacy `log.Print*` functions. |
+|`TF_LOG_SDK` | All SDKs for providers used during the run | Only applies to logs written with `terraform-plugin-log`. For remaining SDKV2 logs, use `TF_LOG` or `TF_LOG_PROVIDER`. |
+`TF_LOG_SDK_PROTO` | [`terraform-plugin-go`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go) | Overrides `TF_LOG_SDK`. |
+| `TF_LOG_SDK_HELPER_SCHEMA`| [`terraform-plugin-sdk/v2`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema) `helper/schema` package | Overrides `TF_LOG_SDK`. Only applies to logs written with `terraform-plugin-log`. For remaining SDKV2 logs, use `TF_LOG` or `TF_LOG_PROVIDER`. |
+|`TF_LOG_SDK_MUX`| [`terraform-plugin-mux`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-mux)| Overrides `TF_LOG_SDK`.
+
+-> **Note**: We recommend setting `TF_LOG=TRACE` when you report bugs to issue trackers. This turns on all loggers at their most verbose setting.
+
+## Logging Levels
+
+To specify logging verbosity, set a [logging environment variable](#logging-environment-variables) to one of the following values. The values are listed in order of least verbose to most verbose:
+
+- **`OFF`** - Turns off logging for that logger.
+- **`ERROR`** - Least verbose, typically provides more detail about user-facing errors.
+- **`WARN`** - Typically includes warnings about deprecations or other environment problems.
+- **`INFO`** - Can include details about run environment or configuration.
+- **`DEBUG`** - Can include additional details about behavior, especially behaviors that may be non-intuitive.
+- **`TRACE`** - Most verbose, and typically includes low-level execution steps.
+
+This example sets the loggers for all providers used in the run to `WARN` verbosity.
+
+```
+TF_LOG_PROVIDER=WARN
+```
+
+-> **Note:** Prior to Terraform v0.15.0, levels besides `TRACE` may not be
+entirely reliable. Refer to [this
+PR](https://github.com/hashicorp/terraform/pull/26632) for more details.
+
+
+## Log Format
+
+By default, logs are written as...
+TODO: what's the difference between default and JSON format?
+
+`JSON` will output logs in a machine-readable format. The format is not guaranteed to be stable between releases.
+TOD: When we pass JSON, what verbosity level will happen?
+
+TODO: Should we say a bit about how logs are written that makes them esier to parse? Are they prepended with their subsystem? Let's say we did TRACE verbosity - how can they tell which logs are relevant to them/filter?
+
```

Comment:
I think diving into this topic is a whole separate section, e.g. "Parsing Logs with jq" -- I think we can probably save this for inclusion later. We'd also need to talk more about the various SDK logging keys available (`tf_provider_addr`, `tf_rpc`, `tf_proto_version`, etc.) too.

```suggestion
```